lib/SILOptimizer/Transforms/DeadObjectElimination.cpp

```diff
@@ -763,6 +763,9 @@ class DeadObjectElimination : public SILFunctionTransform {
   DominanceInfo *domInfo = nullptr;
 
   void removeInstructions(ArrayRef<SILInstruction*> toRemove);
+  
+  void salvageDebugInfo(SILInstruction *alloc, SILInstruction *toBeRemoved);
```

Comment:
Can you add a `///` Doxygen comment here explaining how this function works (i.e., that it's to be called on dead stores to the object and creates a fragment expression)?